### PR TITLE
NEPT-2615: Misleading message when scheduling publications of nodes in draft status

### PR DESCRIPTION
**Pull request by enriquelacoma**
### Description

Update https://www.drupal.org/project/scheduler_workbench/issues/2048999 to include patches/scheduler_workbench-allowed_status.patch and fix behaviour for new nodes

### Change log

- Changed: Patch for issue https://www.drupal.org/project/scheduler_workbench/issues/2048999
- Removed: Local patch patches/scheduler_workbench-allowed_status.patch

### Commands

[Insert commands here]

### Checklist

- [x] Check if there are hook_updates and they are documented
- [x] Add right labels to indicate in the devops ticket the commands to run
- [x] Remove symlinks if it's a module removal

### CODE REVIEW

**Review by drishu — DISMISSED** [review]

File: resources/multisite_drupal_standard.make [hunk 1]
```diff
@@ -661,24 +661,26 @@ projects[rules][patch][] = https://www.drupal.org/files/issues/2019-01-25/302844
 ; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-2160
 projects[rules][patch][] = https://www.drupal.org/files/issues/file_events-826986-31_0.patch
 
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-2615
+; We use the dev version of the module to be able to run the module simpletes.
```
Comment (tomv-be) [inline resources/multisite_drupal_standard.make]: The final 't' in 'simpletest' is missing.

File: resources/multisite_drupal_standard.make [hunk 2]
```diff
@@ -661,24 +661,26 @@ projects[rules][patch][] = https://www.drupal.org/files/issues/2019-01-25/302844
...
 projects[scheduler][subdir] = "contrib"
-projects[scheduler][version] = 1.5
+projects[scheduler][download][type] = "git"
+projects[scheduler][download][url] = "https://git.drupalcode.org/project/scheduler.git"
+projects[scheduler][download][revision] = "89707ba3affa72beea0b428230e61f4c5a0c1283"
 
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-2615
+; We use the dev version of the module to be able to run the module simpletes.
```
Comment (tomv-be) [inline resources/multisite_drupal_standard.make]: Missing 't' at the end of this line as well.

**Review by tomv-be — DISMISSED** [review]

File: resources/multisite_drupal_standard.make (hunk 1 above)
Comment (enriquelacoma) [reply resources/multisite_drupal_standard.make]: Changed to simpletest

File: resources/multisite_drupal_standard.make (hunk 2 above)
Comment (enriquelacoma) [reply resources/multisite_drupal_standard.make]: Changed to simpletest

**Review by drishu — APPROVED** [review]

**Pull request merged**